app/src/evaluation/utils/dataset_mapping.py

```diff
@@ -0,0 +1,37 @@
+"""Utility functions for dataset name mapping."""
+
+from typing import Dict
+
+# Static mapping from CLI dataset IDs to their display names
+# This should match the dataset_label values in get_ingester_config
+DATASET_MAPPING = {
+    "ca_ftb": "CA FTB",
+    "ca_public_charge": "Keep Your Benefits",
+    "ca_wic": "WIC",
+    "covered_ca": "Covered California",
+    "edd": "CA EDD",
+    "irs": "IRS",
+    "la_policy": "DPSS Policy",
+    "ssa": "SSA",
+}
+
+
+def get_dataset_mapping() -> Dict[str, str]:
+    """Get mapping from CLI dataset names to DB dataset names.
+
+    Returns:
+        Dict mapping CLI dataset names (e.g. 'ca_ftb') to DB dataset names (e.g. 'CA FTB')
+    """
+    return dict(DATASET_MAPPING)  # Return a copy to ensure immutability
```

Comment:
https://adamj.eu/tech/2022/01/05/how-to-make-immutable-dict-in-python/#how-to-use-mappingproxytype